src/unit_test.c

```diff
@@ -282,6 +299,12 @@ static int tf_init(struct TestFramework* tf, int argc, char** argv)
             return EXIT_FAILURE;
         }
 
+        /* Check if we need to print help */
+        if (argv[1] && (strcmp(argv[1], "-help") == 0 || strcmp(argv[1], "-h") == 0)) {
```

Comment:
Noticed that `--help` is currently not working, so could add that here, since the double dash variant is explicitly listed in the output. Same for `--print_tests` in the later commit.

Reply:
good catch. Fixed.
Also refactored the code so this doesn't happen again.